### DISCUSSION TITLE
ImportError when running run-docker.sh quicktest

### DISCUSSION THREAD

**Discussion by iyinolu** [Xilinx/finn#1118] [Q&A] [answered]
I am new to FINN and would like to explore creating Quantized Neural Nets for FPGA targets. I run into this issue when running the quicktest as mention in the [documentation](https://finn.readthedocs.io/en/latest/getting_started.html). 

![Screenshot 2024-06-30 075645](https://github.com/Xilinx/finn/assets/47259302/b85366a9-c122-42c1-9533-3885fc806440)

I am running this on Ubuntu via WSL. I'd appreciate any recommendation on how to properly setup my environment to prevent the ImportError as shown on the screenshot. Thank you alot.

**Comment by iyinolu** [comment]
@maltanar Please could you help me with this issue?

**Comment by iyinolu** [comment]
@fpjentzsch Please could you assist me with this?

**Comment by fpjentzsch** [comment, marked as answer]
Hi, please continue the discussion in this duplicate thread: https://github.com/Xilinx/finn/discussions/1121
